### ISSUE TITLE
Cosmos Staking State APR vs Stated APY vs Actual APR vs Actual APY. 

### ISSUE THREAD

**Issue by tolltalcrypto** [shapeshift/web#2102]
### Overview

When looking at the various Cosmos Staking information items, on the overview page when staking ATOM for the first time through the platform it states that the apr is 12%* (screenshot below)

But on the previous page on the Defi Earn page, it shows atom being around 17.03% APY. 

If you click on the ATOM Staking on the earn page once you've staked, it shows the amount staked and the atom APR now is 17.03%. 

If you check Keplr, the APR is closer to 19%~ APR. 

### References and additional details

see screenshots below. 

### Acceptance Criteria

It feels like we should be consistent all the way through on this. If we have APR in one spot we should have it in all, if we have APY in one spot we should have it all.

Not sure what the lift to convert everything for cosmos APRs being brought in to APY's to match the rest of the staking opportunities, but i think we should make an effort to do so.

### Screenshots/Mockups

![image](https://user-images.githubusercontent.com/94986227/176776173-3e50e2a5-b459-49cf-b076-8ecaa87f2613.png)
![image](https://user-images.githubusercontent.com/94986227/176776208-d3aefabb-d8e4-4702-9c1b-4a23c280c46c.png)
![image](https://user-images.githubusercontent.com/94986227/176776272-32c5e8cd-ce56-487e-8300-547c7a53e353.png)
![image](https://user-images.githubusercontent.com/94986227/176776339-d57af903-dda2-4f7d-8007-e985cfe26756.png)

**Comment by gomesalexandre** [comment]
@tolltalcrypto This one has both engineering and product ramifications as to why these values are different

tl;dr is the first "Earn up to x% APY" information is hardcoded and conservative, while the one in e.g Cosmos staking modal is the actual APY, network-fetched. 

It will be trivial (☕ time) to change, this will just have to be changed to the same dynamic data we use in e.g DeFi:

https://github.com/shapeshift/web/blob/b3644065e3a650acf1866aa165d4d6cd6f2fb6a0/src/plugins/cosmos/components/modals/Staking/views/GetStarted/GetStarted.tsx#L48-L51

The question for @shapeshift/product is, do we want to use the dynamic APY data for the "Earn up to" copy? Or do we still want to keep the conservative, hardcoded copy, and change it over time?

**Comment by tolltalcrypto** [comment]
Thanks @gomesalexandre I just noticed it and figured I'd bring it up as I would like to see us be consistent at least throughout a flow. 
Is a calc to turn apr into APY able to be done and applied to other IBC chains that calculate their staking rewards in APR moving forward to "standardize" if that's what we want to do? (probably a product thing again obviously)

**Comment by gomesalexandre** [comment]
> Is a calc to turn apr into APY able to be done and applied to other IBC chains that calculate their staking rewards in APR moving forward to "standardize" if that's what we want to do? (probably a product thing again obviously)

That's a good point! The current logic we have basically considers APR and APY to be the same, so as rightly noticed, what we show as "APY" in DeFi is actually the APR data. We could write some small utils that would go from one to the other and that would be fairly low effort, so that what we show as APY is really APY.

I'm not sure about the product value, as the difference should be minimal (far from the 5ish percent different from these screenshots), but if that's something we want for consistency, happy to spec it!

**Comment by gomesalexandre** [comment]
Another one for @shapeshift/product: do we want to change the "Estimated returns" copy to "Actual returns" in the Cosmos staking modal, given we can get the actual APY vs. the hardcoded one? See:

https://github.com/shapeshift/web/issues/2111#issuecomment-1176617590
